Library/Formula/napi.rb

```diff
@@ -0,0 +1,33 @@
+class Napi < Formula
+  desc "Napi Projekt Client and Subotage subtitle format converter both written in bash"
```

Comment:
Could you shorten the description? We try to keep “<name>: <desc>” under 80 chars, it’s 85 here. You could e.g. remove “both written in bash”. Thanks!